src/main/scala/arcana/death/Repl.scala

```diff
@@ -0,0 +1,38 @@
+package arcana.death
+
+import scala.io.StdIn
+
+class Repl(options: ReplOptions) {
+  import Repl._
+  printf(
+    "\u001b[32;1m" +
+      "Welcome to Arcana %s \n" +
+      "Type in expressions for evaluation.\n" +
+      "Type in exit or ctrl-d to quit\n" +
+      "\u001b[0m",
+    version
+  )
+  parse(true)(options)
+
+}
+
+object Repl {
+  private val version = "0.0"
+
+  private def parse(continue: Boolean)(implicit options: ReplOptions): Unit = {
```

Comment:
Why not just remove the `implicit` and move the function to the `class Repl` so it can refer to `options: ReplOptions` directly.